WcaOnRails/spec/views/registrations/psych_sheet_event.html.erb_spec.rb

```diff
@@ -23,7 +23,7 @@
     newcomer2 = FactoryGirl.create(:user, name: "Newcomer II")
 
     [the_best, tied_first, tied_second, newcomer1, newcomer2].each do |user|
-      FactoryGirl.create(:registration, :accepted, user: user, competition: competition, event_ids: ["333"])
+      FactoryGirl.create(:registration, :accepted, user: user, competition: competition, events: [Event.find("333")])
```

Comment:
I couldn't find a way to make it work with my modification, so I chose to pass an `Event` instead of a `String`.
I actually didn't quite understand how the FactoryGirl could create the Event from an array of ids before :s